app/views/layouts/application.html.erb

```diff
@@ -28,11 +28,14 @@
 <body>
 <%= render 'shared/skip_links' %>
 <%= render 'shared/header' %>
-<%= render 'shared/alerts' %>
 <%= yield :breadcrumbs %>
 
 <main role="main" id="main">
   <div class='l-constrained'>
+    <div class="l-flashes">
```

Comment:
Can we move this div into the partial, or make it a conditional somehow? Just thinking that we were setting the tone for how to do flash messages across MAS and having an empty div when there are no flash messages seems like it could be improved